cl/citations/management/commands/cl_find_citations_recap.py

```diff
@@ -13,28 +13,41 @@
 from cl.lib.celery_utils import CeleryThrottle
 from cl.lib.command_utils import VerboseCommand
 from cl.lib.types import OptionsType
-from cl.search.models import Opinion
+from cl.search.models import Opinion, RECAPDocument
+
+
+def _parse_args_list(args: str) -> list:
+    return [s.strip() for s in args.split(",")]
+
+
+# def get_documents():
 
 
 class Command(VerboseCommand):
     help = "Parse citations from RECAP documents."
 
     def add_arguments(self, parser: CommandParser) -> None:
         parser.add_argument(
-            "--doc-id",
-            type=int,
+            "--doc-pks",
+            type=_parse_args_list,
             nargs="*",
-            help="ids of citing documents",
+            help="Database pks of citing document. If this is specified, will ignore all other filters.",
+        )
+        parser.add_argument(
+            "--doc-ids",
+            type=_parse_args_list,
+            nargs="*",
+            help="IDs of documents in PACER. If this is specified, will ignore all other filters.",
```

Comment:
doc-pks vs doc-ids is really confusing to me. Maybe you mean pacer_doc_ids? Do we need this? Feels like YAGNI?

Reply:
It seemed reasonable to me that one might want to extract citations from xyz documents from particular case, whose PACER doc IDs are viewable but whose PKs would require lookup. But yeah, IDK when you guys typically run these commands outside of the normal scheduled task process. I'll remove it.